pkg/datapath/linux/ipsec/ipsec_linux_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
```

Comment:
needs sign-off

Reply:
Ping @jrfastab, could you add SOB to this one commit so we can merge? Thx

Reply:
Ping again ;)

Reply:
added now. Finally made it to the top of my stack.